python-avd/pyavd/_eos_designs/structured_config/network_services/vxlan_interface.py

```diff
@@ -24,152 +26,91 @@ class VxlanInterfaceMixin(Protocol):
     """
 
     @cached_property
-    def vxlan_interface(self: AvdStructuredConfigNetworkServicesProtocol) -> dict | None:
+    def _multi_vtep(self: AvdStructuredConfigNetworkServicesProtocol) -> bool:
+        return self.shared_utils.mlag is True and self.shared_utils.evpn_multicast is True
+
+    @structured_config_contributor
+    def vxlan_interface(self: AvdStructuredConfigNetworkServicesProtocol) -> None:
         """
-        Returns structured config for vxlan_interface.
+        Update structured config for vxlan_interface.
```

Comment:
We have used "Set structured ...." in other places. Also for methods, I think we mostly use update_<> when we pass an object that is inplace updated. Maybe we need a naming guide...